Svenska Dagbladet.js

```diff
@@ -0,0 +1,247 @@
+{
+	"translatorID": "7adba17c-7e98-4a13-8325-e19383b09eab",
+	"label": "Svenska Dagbladet",
+	"creator": "Sebastian Berlin",
+	"target": "^https://www\\.svd\\.se/(sok\\?)?",
```

Comment:
if the last element has a ?, you don't need it. I'd just use  `^https://www\\.svd\\.se/.`, i.e. every page on the site other than the homepage.

Reply:
True, removed that.